observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc

```diff
@@ -0,0 +1,41 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="observability-ui-plugins-overview"]
+= Observability UI plugins overview
+include::_attributes/common-attributes.adoc[]
+:context: observability-ui-plugins-overview
+
+toc::[]
+
+:FeatureName: The {coo-full}
+include::snippets/technology-preview.adoc[leveloffset=+2]
+
+You can use the {coo-first} to install and manage UI plugins to enhance the observability capabilities of the {product-title} web console.
+The plugins extend the default functionality, providing new UI features for monitoring, troubleshooting, distributed tracing, and cluster logging.
+
+["dashboards_{context}"]
+== Dashboards
+
+The dashboard UI plugin supports enhanced dashboards in the {product-title} web console at *Observe* -> *Dashboards*.
+You can add other Prometheus data sources from the cluster to the default dashboards, in addition to the in-cluster data source.
+This results in a unified observability experience across different data sources.
+
+["troubleshooting_{context}"]
+== Troubleshooting
+
+The troubleshooting panel UI plugin provides observability signal correlation, powered by the open source Korrel8r project.
+You can use the troubleshooting panel available from the *Observe* -> *Alerting* page to easily correlate metrics, logs, alerts, netflows, and additional observability signals and resources, across different data stores.
```

Comment:
@gabriel-rh The troubleshooting panel can be also triggered from the masthead application launcher, as seen in this PR: https://github.com/openshift/troubleshooting-panel-console-plugin/pull/67/files. This is enabled form 4.17. Would you think this is something we can add in this paragraph?

Reply:
I think this is already included here: https://github.com/openshift/openshift-docs/pull/81463/files#diff-e8d2db0b08d6c6517eb6812f32c2a129172a3511808372eda637b1046931505bR13

Reply:
Added here as well